File: c++/simple_Interest.cpp
#include<iostream>
using namespace std;
class simple_Interest{
	int p,i,t,ans;
	public:
		void input()
		{
			cout<<"Enter Principle"<<endl;
			cin>>p;
			cout<<"Enter Interest"<<endl;
			cin>>i;
			cout<<"Enter Time"<<endl;
			cin>>t;
		}
		void cal()
		{
			ans=(p*i*t)/100;
		}
		void Display()
		{
			cout<<"Simple interest is="<<ans;
		}
};
int main()
{
	simple_Interest s;
	s.input();
	s.cal();
	s.Display();
}

File: java_assinment 1/simpleinterest.java
import java.util.*;
public class simpleinterest
{
float si ,principalamount, rate, time;
void calculate()
{
System.out.println("Enter value of principalamount, rate, time");
Scanner sc=new Scanner(System.in);
principalamount=sc.nextFloat();
rate=sc.nextFloat();
time=sc.nextFloat();
si=(principalamount*rate*time)/100;
System.out.println("Simple Interest="+si);
}
public static void main(String[] args)
{
simpleinterest si=new simpleinterest();
si.calculate();
}
}
 
